blueprints/acceptance-test/files/tests/acceptance/__name__-test.js

```diff
@@ -17,14 +17,14 @@ describe('Acceptance: <%= classifiedModuleName %>', function() {
   });
 
   afterEach(function() {
-    Ember.run(application, 'destroy');
+    application.destroy();
   });
 
-  it('can visit /<%= dasherizedModuleName %>', function() {
+  beforeEach(function() {
```

Comment:
It's fairly common to separate setup concerns. In this case, application setup vs page navigation, but what I'd probably do IRL, is either extract the app setup into its own helper, or If I was feeling like a BDD purist that day, I'd introduce a new description coupled with the new setup. E.g.

```
describe('Acceptance: <%= classifiedModuleName %>', function() {
  beforeEach(function() {
    this.application = startApp();
  });

  afterEach(function() {
    this.application.destroy();
  });

  describe("visiting /<%= dasherizedModuleName %>", function() {
    beforeEach(function() {
      visit('/<%= dasherizedModuleName %>');
    });

    it("transitions successfully", function() {
      expect(currentPath()).to.equal('<%= dasherizedModuleName %>');
    });
  });
});
```

I always chafe at the amount of boilerplate in the acceptance tests anyhow, what about introducing a dedicated test helper for acceptance tests, so it could become something like:

```
describeAcceptance(' <%= classifiedModuleName %>', function() {
  beforeEach(function() {
    visit('/<%= dasherizedModuleName %>');
  });

  it("transitions successfully", function() {
    expect(currentPath()).to.equal('<%= dasherizedModuleName %>');
  });
});
```